esphome/const.py

```diff
@@ -470,6 +472,7 @@
 ICON_FLOWER = 'mdi:flower'
 ICON_GAS_CYLINDER = 'mdi:gas-cylinder'
 ICON_GAUGE = 'mdi:gauge'
+ICON_IMPEDANCE = 'mdi:omega'
```

Comment:
```suggestion
ICON_OMEGA = 'mdi:omega'
```

Icons are always named after their icon name - they could for example be used in many different circumstances.